deps/base64/base64.gyp

```diff
@@ -84,6 +85,14 @@
           'xcode_settings': {
             'OTHER_CFLAGS': [ '-mssse3' ]
           },
```

Comment:
This file is now removed on main branch.

Reply:
removed base64 patches